ern-cauldron-api/src/gitstore.js

```diff
@@ -10,17 +10,33 @@ import path from 'path'
 
 const CAULDRON_FILENAME = 'cauldron.json'
 
+export type TypeCauldronCodePushMetadata = {
```

Comment:
it's more of a code push meta that happens to be stored in cauldron. It would be nicer to just say `CodePushMetadata` ? it's implied that flow type . `cauldron` in the name can be avoided?

Reply:
agreed. will do